label_studio/core/api_permissions.py

```diff
@@ -4,3 +4,10 @@
 class HasObjectPermission(BasePermission):
     def has_object_permission(self, request, view, obj):
         return obj.has_permission(request.user)
+
+
+class HasOwnerPermission(BasePermission):
```

Comment:
Are soft deletes exclusive to Owner, or would this be something that an Admin would also be capable of?

Reply:
This permission handler is native to LS where there are no roles.  In LSE you're right I'm using LSE's permission handler that handles admins as well (but I'll double check to be sure)